composer.lock

```diff
@@ -1483,11 +1901,14 @@
                 }
             },
             "autoload": {
-                "psr-0": {
-                    "Google_": "src/"
```

Comment:
I think we have a number of places where we are still using the old PSR-0 `Google_`... class names. I guess the aliases file makes this backwards compatible? Can you confirm that all of our Google client functionality that is using the underscore separated global class names still works?